.github/workflows/cd.yml

```diff
@@ -19,10 +19,10 @@ jobs:
     runs-on: ubuntu-latest
```

Comment:
I normally like to hardcode the version. But could you please let me know the rational of using latest here :) ?

Reply:
True we should do it for builds.

Normally, CD is inconsequential with multi-platform with docker `qemu` + `buildx`. However, since we're not building on image layer and including the binary directly we're not benefiting from it.

- [ ] I think what we need is for `cd.yml` to utilize building on docker engine so we can utilize docker qemu.
- [ ] While for `ci.yml` our builds should run on multi-platform via a matrix targeting different versions.

@DieHard073055 can you open 2 separate issues for these 2 problems.

Reply:
I hope i understood everything.
https://github.com/DeFiCh/metachain/issues/61
https://github.com/DeFiCh/metachain/issues/62